.travis.yml

```diff
@@ -308,14 +308,15 @@ install:
       wget https://github.com/mozilla/geckodriver/releases/download/$GECKODRIVER_VERSION/geckodriver-$GECKODRIVER_VERSION-linux64.tar.gz -O geckodriver.tar.gz;
       mkdir bin;
       tar zxf geckodriver.tar.gz -C bin;
+      git clone -b search-by-extent-postgis https://github.com/mtnorthcott/postgis-docker.git;
```

Comment:
Is this still necessary?

Reply:
Not necessary once the change in https://github.com/GeoNode/postgis-docker/pull/2 is available in the Docker hub.

Reply:
@mtnorthcott pr mrged and docker image rebuilt, can we remove this one please?